mmdet3d/models/roi_heads/bbox_heads/multi_group_head.py

```diff
@@ -733,14 +729,8 @@ def get_task_detections(self, num_class_with_bg, batch_cls_preds,
             box_preds = box_preds.float()
             cls_preds = cls_preds.float()
 
-            if self.encode_background_as_zeros:
-                # this don't support softmax
-                assert self.use_sigmoid_score is True
-                # total_scores = torch.sigmoid(cls_preds)
-                total_scores = cls_preds
-            else:
-                # encode background as first element in one-hot vector
-                total_scores = torch.sigmoid(cls_preds)[..., 1:]
+            # total_scores = torch.sigmoid(cls_preds)
+            total_scores = cls_preds
```

Comment:
If they are the same, we may skip this assignment and only use `cls_preds` in the following code